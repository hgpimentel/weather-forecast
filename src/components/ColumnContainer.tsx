import styled from "styled-components";

const ColumnContainer = styled.div`
  display: flex;
  flex-direction: column;
`;

export default ColumnContainer;
