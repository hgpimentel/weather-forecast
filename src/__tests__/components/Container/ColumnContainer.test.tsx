import renderer from "react-test-renderer";
import ColumnContainer from "../../../components/Container/ColumnContainer";

describe("Column container component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<ColumnContainer />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
