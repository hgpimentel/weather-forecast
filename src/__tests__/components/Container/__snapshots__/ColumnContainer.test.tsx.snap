// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column container component renders correctly 1`] = `
<div
  className="sc-bdnxRM eoiijl"
/>
`;
